core/src/main/java/org/elasticsearch/search/suggest/term/TermSuggestionBuilder.java

```diff
@@ -48,6 +51,9 @@ public TermSuggestionBuilder(String name) {
         super(name, "term");
```

Comment:
I mentioned the NamedWritable#getWriteableName() above, I think "term" here should be a constant that identifies the type of SuggestionBuilder this is. Then we don't need it as an argument in the super constructor anymore.